package.json

```diff
@@ -11,6 +11,7 @@
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
     "classnames": "^2.3.2",
+    "jest": "^27.5.1",
```

Comment:
It would be nice to have `cypress` to fully complete the task 🙂
https://www.cypress.io/

Reply:
Added